x-pack/test/licensing_plugin/apis/changes.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../services';
+import { SecurityService } from '../../common/services';
+import { PublicLicenseJSON } from '../../../plugins/licensing/server';
+
+const delay = (ms: number) => new Promise(res => setTimeout(res, ms));
+
+export default function({ getService, getPageObjects }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esSupertestWithoutAuth = getService('esSupertestWithoutAuth');
+  const security: SecurityService = getService('security');
+  const PageObjects = getPageObjects(['common', 'security']);
+
+  const scenario = {
+    async setup() {
+      await security.role.create('license_manager-role', {
+        elasticsearch: {
+          cluster: ['all'],
+        },
+        kibana: [
+          {
+            base: ['all'],
+            spaces: ['*'],
+          },
+        ],
+      });
+
+      await security.user.create('license_manager_user', {
+        password: 'license_manager_user-password',
+        roles: ['license_manager-role'],
+        full_name: 'license_manager user',
+      });
+
+      // ensure we're logged out so we can login as the appropriate users
+      await PageObjects.security.logout();
+      await PageObjects.security.login('license_manager_user', 'license_manager_user-password');
+    },
+
+    async teardown() {
+      await security.role.delete('license_manager-role');
```

Comment:
we don't call teardown in `after` because `security` needs to update its internal status after
https://github.com/elastic/kibana/pull/51818/files#diff-5e8ddd0044ee621101208f9a3b37b755R124
and https://github.com/elastic/kibana/pull/51818/files#diff-5e8ddd0044ee621101208f9a3b37b755R87